src/history/hs_rec.c

```diff
@@ -207,54 +113,48 @@ __hs_insert_record_with_btree(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BT
      * timestamp. Otherwise the newly inserting history store record may fall behind the existing
      * one can lead to wrong order.
      */
-    WT_ERR_NOTFOUND_OK(
-      __wt_hs_cursor_position(session, cursor, btree->id, key, tw->start_ts, srch_key), true);
+    cursor->set_key(cursor, 4, btree->id, key, tw->start_ts, UINT64_MAX);
```

Comment:
4?

Reply:
It's the number of arguments after the first fixed cursor argument.